systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -994,9 +992,6 @@ void testDynamicallySetMM2LoggingLevels(ExtensionContext extensionContext) {
         resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(clusterName + "-source", 3).build());
         resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(clusterName + "-target", 3).build());
         resourceManager.createResource(extensionContext, false, KafkaClientsTemplates.kafkaClients(false, kafkaClientsName).build());
```

Comment:
I guess that you don't even need to create the clients pod - or you can exchange the `kafkaMM2PodName` in the `execInPod` methods with the clients pod.

Reply:
I would rather not change test code in this PR. Just remove dead/obsolete code.

Reply:
It was connected to deleted `kafkaClientsPodName` on line `998`